docs/SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Current version only.
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability, [please send an on-site message to RAdmin](https://retroachievements.org/user/RAdmin).
```

Comment:
```suggestion
Here's how you can report a security issue:

1. **Do not report security issues in GitHub issues, discussions, or on Discord.**
2. **Send a direct on-site message.** If you believe you've found a security vulnerability, [please send an on-site message to RAdmin](https://retroachievements.org/user/RAdmin).

In your report, please include:

- A brief description of the issue.
- Steps to reproduce the issue, if possible.
- Potential impact of the issue.

We will acknowledge your report within 48 hours of receiving it. We appreciate your help, and we will strive to keep our site as secure as possible. Thank you.
```

Added what *not* to do, as well as what to include in one of these disclosure messages.